python/GafferUI/PathListingWidget.py

```diff
@@ -61,7 +61,7 @@ class PathListingWidget( GafferUI.Widget ) :
 	IconColumn = _GafferUI.IconPathColumn
 
 	## A collection of handy column definitions for FileSystemPaths
-	defaultNameColumn = StandardColumn( "Name", "name" )
+	defaultNameColumn = StandardColumn( "Name", "name", flags = GafferUI.PathColumn.Flags.ResizeStretch )
```

Comment:
I'm not 100% sure about this - here are two downsides from the File Open dialogue that we might want to consider :

- If you have a name that is too wide to see, you can no longer resize the Name column to see it.
- Resizing the Owner column feels kindof weird - you drag the right edge to the right and it stays put while the left edge moves to the left.